include/engine/polyline_compressor.hpp

```diff
@@ -13,14 +15,42 @@ namespace engine
 namespace detail
 {
 constexpr double POLYLINE_PRECISION = 1e5;
```

Comment:
Okay, yeah, I'll just update the variable name and adjust the docs to say that we're only dealing with precision 6 for the encoding and not the decoding 😄